test/test.config.js

```diff
@@ -12,7 +12,7 @@ config.karma.suites['bedrock-vue-create-persona-wizard'] =
 // TODO: consider moving to `bedrock-karma`?
 config.karma.config.webpack.resolve = {
   alias: {
-    "quasar-framework$": path.resolve(
-      __dirname, 'node_modules/quasar-framework/dist/umd/quasar.mat.umd.js')
+    'quasar-$': path.resolve(
```

Comment:
Is this trailing dash correct? Should it be `quasar$`?

Reply:
I really don't know actually I guess we are importing as quasar so the dash might break things. strange that this this did not actually break anything that I noticed.